InformationScripting/src/wrappers/NodeApi.cpp

```diff
@@ -28,15 +28,38 @@
 
 #include "../graph/PropertyMap.h"
 #include "../graph/InformationNode.h"
+#include "../graph/InformationEdge.h"
+#include "../graph/Graph.h"
 
 namespace InformationScripting {
 
 using namespace boost::python;
 
+// The following macro has unused local typdef so we disable the warning for this.
```

Comment:
We could use an extra compiler for this as described here:
http://stackoverflow.com/a/14608296